.github/workflows/deny.yml

```diff
@@ -0,0 +1,65 @@
+name: Deny - Linux
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏haven't always been consistent with it but I'm aiming to steer us toward having a doc comment at the top of the workflow files to describe what it does and the specifics of when it gets evoked (see Component Features for an example).